aws_lambda_builders/__main__.py

```diff
@@ -46,6 +49,22 @@ def _error_response(request_id, http_status_code, message):
     })
 
 
+def _parse_version(version_string):
+
+    if VERSION_REGEX.match(version_string):
+        return float(version_string)
+    else:
+        raise ValueError("Protocol Version does not match : {}".format(VERSION_REGEX.pattern))
+
+
+def version_compatibility_check(version):
+    if _parse_version(lambda_builders_protocol_version) < version:
```

Comment:
This should be `<=` - you want "0.2 == 0.2" to be a pass

Reply:
No this is a condition check when an error is thrown (Incompatible version), I have added func tests to show it works.

(current lamdda builders version < lambda builder version from client)

0.2 < 0.2 comparison will fail, therefore we dont throw a value Error saying incompatible version.
0.2 < 0.3 comparison will pass, throwing a ValueError
0.2 < 0.1 comparison will fail, therefore we dont throw a value Error saying incompatible version

Reply:
Ah okay, I didn't realize this was the case. Thanks for clarification. Can you add a quick comment explaining why this is the case